impeller/renderer/surface.h

```diff
@@ -26,7 +26,7 @@ class Surface {
 
   bool IsValid() const;
 
-  const RenderTarget& GetTargetRenderPassDescriptor() const;
+  RenderTarget& GetTargetRenderPassDescriptor();
```

Comment:
This is perplexing. How would mutating the render target of a surface work?

Reply:
Discussed offline -- this was my (overly) aggressive aversion to copying striking again. ;)

Reply:
Done.